samples/tab-app-monetization/nodejs/ClientApp/src/components/tab.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { app, monetization } from "@microsoft/teams-js";
+import { Text, Button, Flex } from '@fluentui/react-northstar'
+import './tab.css'
+
+const Tab = () => {
+
+    React.useEffect(() => {
+        app.initialize();
+    }, [])
+
+
+    var planInfo = {
+        planId: "<Plan id>", // Plan Id of the published SAAS Offer
```

Comment:
How to create SAAS offer => Please add steps or link in Readme file.

Reply:
Yes we have provided steps in readme for creating SAAS offer